examples/input.rs

```diff
@@ -0,0 +1,62 @@
+extern crate tetra;
+
+use tetra::glm::Vec2;
+use tetra::graphics::{self, Color};
+use tetra::graphics::text::{Text, Font};
+use tetra::input::{self, Key};
+use tetra::{Context, ContextBuilder, State};
+
+struct GameState {
+    input: String,
+    text: Text,
+}
+
+impl GameState {
+    fn update_text(&mut self) {
+        self.text = Text::new(
+            self.input.as_str(),
+            Font::default(),
+            16.0,
+        );
+    }
+}
+
+impl State for GameState {
+    fn update(&mut self, ctx: &mut Context) {
+        if input::is_key_pressed(ctx, Key::Backspace) {
+            self.input.pop();
+            self.update_text();
```

Comment:
There's a `set_content` method on `Text`, which will save you from having to recreate the object every time you update it.